catalog/task/skopeo-inspect/0.1/samples/sample_skopeo-inspect_TaskRun.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: skopeo-inspect-run-empty-params
+spec:
+  params:
+    - name: imageURL
+      value: ""
```

Comment:
Do we want this? I think that if you don't specify any value here (remove the field) the `imageURL` would become mandatory and fail if it's not passed. You wouldn't need to check it yourself.

Reply:
I am confused as to what you are suggesting here. Every single task we have in this repo has a sample/TaskRun.yaml file with all of the parameters set to `""` value

Reply:
I got totally confused. I scrolled too fast I think and I thought this was the actual Task. All good.